src/primitives/cuboid.js

```diff
@@ -3,15 +3,14 @@ import {
 	InstancedBufferAttribute,
 	InstancedBufferGeometry,
 	Mesh,
-	ShaderMaterial,
-	UniformsLib
+	RawShaderMaterial
 } from '../../vendors/three.js';
 
 import { get2PopulatedCoordinateBuffers } from '../bufferUtils.js';
 
 // See https://reference.wolfram.com/language/ref/Cuboid
 // for the high-level description of what is being rendered.
-export default function ({ color = [1, 1, 1], coords, edgeForm = {}, opacity = 1 }, extent) {
+export default function ({ color = [1, 1, 1], coords, edgeForm = {}, opacity = 1 }, uniforms, extent) {
```

Comment:
Ok so uniforms seems to be added across the board. So maybe some documentation on what's up here would be nice. (I won't repeat these comments very time it comes up below).

Reply:
I'll add a comment about it in primitives/index.js

Reply:
Done.